app/src/main/java/com/example/memeapp/presentation/fragment/MemeFragmentDetail.kt

```diff
@@ -0,0 +1,56 @@
+package com.example.memeapp.presentation.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import com.example.memeapp.data.service.implementation.MemesServiceImpl
+import com.example.memeapp.databinding.MemeCardDetailLayoutBinding
+import com.example.memeapp.domain.usecase.GetSingleMemeUseCaseImpl
+import com.example.memeapp.presentation.mvp.contract.FragmentDetailContract
+import com.example.memeapp.presentation.mvp.model.MemeDetailModel
+import com.example.memeapp.presentation.mvp.presenter.MemeDetailPresenter
+import com.example.memeapp.presentation.mvp.view.MemeDetailView
+
+class MemeFragmentDetail : DialogFragment () {
+    private lateinit var presenter: FragmentDetailContract.Presenter
+    private lateinit var binding: MemeCardDetailLayoutBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+
+    ): View {
+        binding = MemeCardDetailLayoutBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        presenter = MemeDetailPresenter(
+            MemeDetailModel(
+                GetSingleMemeUseCaseImpl(MemesServiceImpl())
+            ), MemeDetailView(this, binding)
```

Comment:
this should be formated as
MemeDetailPresenter(
       MemeDetailModel( GetSingleMemeUseCaseImpl(MemesServiceImpl()) ), 
       MemeDetailView(this, binding)

Reply:
Done thx